platform/README.adoc

```diff
@@ -18,11 +18,27 @@ Requirements:
 - https://nodejs.org/en/download/ (Node.js 12.x or above)
 - (optional) access to a running OpenShift instance
 
+== Development 
```

Comment:
Running should be first and then development

Reply:
Putting it at the end of the file as we have other running sections e.g as mobile apps, running on openshift etc